contracts/FIFOMarket.sol

```diff
@@ -71,6 +71,7 @@ contract FIFOMarket is
     return _queueNextInsertIndex - _queueHeadIndex;
   }
 
+  // todo do we want this to report NRTs in either or wei? because removal token balances are denominated in wei
```

Comment:
I think wei as we do this for every unit-- doing otherwise might just be confusing as then two possible units are returned from different functions

Reply:
ok cool, makes the most sense to me